packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -677,13 +677,19 @@ export function attach(
     operations[i++] = rendererID;
     operations[i++] = currentRootID; // Use this ID in case the root was unmounted!
     operations[i++] = 0; // No strings to send.
+
+    const mostRecentlyInspectedElementID = mostRecentlyInspectedElement?.id;
     updatedIDs.forEach(fiberId => {
       const {errors = [], warnings = []} = errorsOrWarnings.get(fiberId) ?? {};
 
       operations[i++] = TREE_OPERATION_UPDATE_ERRORS_OR_WARNINGS;
       operations[i++] = fiberId;
       operations[i++] = errors.length;
       operations[i++] = warnings.length;
+
+      if (fiberId === mostRecentlyInspectedElementID) {
+        hasElementUpdatedSinceLastInspected = true;
+      }
```

Comment:
👍

Reply:
I almost gave up on that one. Without clear variable names I wouldn't have know :+1: